src/registrar/templatetags/custom_filters.py

```diff
@@ -48,3 +51,18 @@ def contains_checkbox(html_list):
         if re.search(r'<input[^>]*type="checkbox"', html_string):
             return True
     return False
+
+
+@register.filter
+def get_organization_long_name(organization_type):
+    # https://gist.github.com/OmenApps/3eef60ba4204f3d1842d9d7477efcce1#file-django_choices-txt-L28
```

Comment:
(blocking) This link is broken for me... I'd also say that this link needs a bit more context (brief explanation as to what the link is).

If the link is no longer needed you can delete this, also